cymetric/execution.py

```diff
@@ -30,33 +30,41 @@ def __init__(self, name):
         self.conds = []
 
     def __call__(self, *args, **kwargs):
+        """Checks that column name is callable."""
```

Comment:
I would say something like "stub function that prevents ColumnProxies from being called."